README.md

```diff
@@ -1,2 +1,5 @@
 # hello-world
 this is the edit and commit test
+
```

Comment:
Dies ist ein Kommentar den man direkt in den Code schreiben kann um sachen im Kontext zu kommentieren :)